dev-python/asynctest/asynctest-0.12.2.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Enhance the standard unittest with features for testing asyncio libraries"
+HOMEPAGE="https://github.com/Martiusweb/{$PN}/ https://pypi.org/project/${PN}/"
```

Comment:
No variables inside HOMEPAGE. Also, you're got typo there.

Reply:
Spelled the "asynctest" out in the homepage. I did not know using variables there was against the policy.